sprite/test.js

```diff
@@ -0,0 +1,24 @@
+document.body.appendChild(html`
+<section id="sprite">
+  <h2>Sprite</h2>
+
+  <!-- Default inline toaster -->
+  <div class="test-container">
+    <span>Sprite</span>
+  </div>
+
+</section>
+`)
+
+// TODO write tests
+
+function html ([str, ...strings], ...values) {
```

Comment:
this helper functions appears to be at the bottom of all the tests, could it not be sourced?

Reply:
Willfix